### PR TITLE
c31, c32 and updated c33

### PR DESCRIPTION
**Pull request by ScrewThisNoise**

### CODE REVIEW

File: translation/en/RedirectedResources/assets/abdata/communication/info_17/communication_off_33/translation.txt [hunk 1]
```diff
@@ -1,551 +1,551 @@
...
-//「どうしたの？　[Hあだ名]」=
-//「好きよ」=
-//「ふふ、今だけは独り占めしちゃってもいいのよね？　[Pあだ名]」=
-//「ねぇ、今から愛を育まない？ [Pあだ名]」=
-//「あら、時間があるなら話さない？」=
-//「あら、[Pあだ名]。時間があるなら話さない？」=
-//「もちろん。歓迎するよ！」=
-//「ふふ、嬉しいわ。よろしくお願いするわね、部長さん？」=
+「あら、ここ女子トイレよ？」=Oh my, this is the girl's toilet, you know?
+「女の子の生態でも調べにきたの？ くすくすっ」=Came to check out even girls' private territory? *Giggle*
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/communication/info_17/communication_off_33/translation.txt:2]: Looks like spaces got changed in the jp text

File: translation/en/RedirectedResources/assets/abdata/communication/info_25/communication_off_33/translation.txt [hunk 2]
```diff
@@ -1,551 +1,551 @@
...
-//「どうしたの？　[Hあだ名]」=
-//「好きよ」=
-//「ふふ、今だけは独り占めしちゃってもいいのよね？　[Pあだ名]」=
-//「ねぇ、今から愛を育まない？ [Pあだ名]」=
-//「あら、時間があるなら話さない？」=
-//「あら、[Pあだ名]。時間があるなら話さない？」=
-//「もちろん。歓迎するよ！」=
-//「ふふ、嬉しいわ。よろしくお願いするわね、部長さん？」=
+「あら、ここ女子トイレよ？」=Oh my, this is the girl's toilet, you know?
+「女の子の生態でも調べにきたの？ くすくすっ」=Came to check out even girls' private territory? *Giggle*
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/communication/info_25/communication_off_33/translation.txt:2]: Looks like spaces got changed in jp text

File: translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_31/translation.txt [hunk 3]
```diff
@@ -1,119 +1,47 @@
...
+早速やろう=Let's get to it.
+じゃあ帰ろう=Then, let's go home.
 やろう=Let's do it.
 ちょっと待ってて=Wait a bit.
-嬉しいよ=I'm glad.
-うーん……=Er...
-お願いします！=I'll leave it to you!
-シたい=I want to.
-
-//OPTION[1]:うーん、そうだね。少し怖いかな=
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_31/translation.txt:49]: Options got removed?

File: translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_32/translation.txt [hunk 4]
```diff
@@ -1,111 +1,39 @@
...
+もっと君のことを知りたい=I want to know more about you.
+少し恥ずかしいな=It's a bit embarrassing.
+いいよ=Sure.
+……ごめん=...Sorry.
+ごめん=Sorry.
+そうだね=Indeed.
+うーん……=Well...
 もちろん=Of course.
-
-//OPTION[1]:うん=
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_32/translation.txt:41]: Removed options

**Review by ManlyMarco — COMMENTED** [review]
Found some potential issues. I didn't check inside every file, but it looks like there are no other whitespace changes, they were isolated to the first commit in the PR (the fix one).

File: translation/en/RedirectedResources/assets/abdata/communication/info_17/communication_off_33/translation.txt (hunk 1 above)
Comment (ScrewThisNoise) [reply translation/en/RedirectedResources/assets/abdata/communication/info_17/communication_off_33/translation.txt:2]: re-dump. Old dump for personality failed to load text ingame due to problems with spaces

File: translation/en/RedirectedResources/assets/abdata/communication/info_25/communication_off_33/translation.txt (hunk 2 above)
Comment (ScrewThisNoise) [reply translation/en/RedirectedResources/assets/abdata/communication/info_25/communication_off_33/translation.txt:2]: re-dump. Old dump for personality failed to load text ingame due to problems with spaces

File: translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_32/translation.txt (hunk 4 above)
Comment (ScrewThisNoise) [reply translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_32/translation.txt:41]: Fixed

File: translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_31/translation.txt (hunk 3 above)
Comment (ScrewThisNoise) [reply translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_31/translation.txt:49]: Fixed

**Review by ManlyMarco — APPROVED** [review]

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/15/6/translation.txt [hunk 5]
```diff
@@ -1,3 +1,3 @@
-//「付いてこい……だ？　何処に、何の目的で行くんだ？」=
-//「いやそれは、まだ秘密という事で……サプライズ的な？」=
-//「怪しさしかないが……まぁいいだろう。おまえに釣られてみるというのも、面白そうだしな」=
+「付いてこい……だ？　何処に、何の目的で行くんだ？」="""Follow me""... you say? Where, and for what reason?"
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/15/6/translation.txt:1]: Quote issues

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/15/70/translation.txt [hunk 6]
```diff
@@ -1,6 +1,6 @@
...
-//「このまま一緒にやっていたら、二人で学園トップの秀才になっちゃうかもね」=
-//「おまえはすぐそうして調子に乗る。頭が良くても、賢くなければ意味がないのだぞ？」=
-//「ははっ、なんかお母さんみたいな事言われちゃったね」=
-//「お、おかっ！？　そこはせめて、おねーさんと言え！」=
+「二人でやると、勉強も楽しいね」=When we do it together, even studying is fun, don't you think?
+「あぁ、否定はできないな。効率も上がるし、良いこと尽くめという訳か」=Yeah, I can't refute that. The efficiency goes the roof too, nothing but good stuff.
+「このまま一緒にやっていたら、二人で学園トップの秀才になっちゃうかもね」=If we keep doing it together like this, we might end up becoming the top prodigies in this school.
+「おまえはすぐそうして調子に乗る。頭が良くても、賢くなければ意味がないのだぞ？」=You always get carried away at the drop of a hat like that. Even if you have a good head, if you don't have street smarts, there's no point, you know?
+「ははっ、なんかお母さんみたいな事言われちゃったね」=Haha--, you sound like you're my mother.
+「お、おかっ！？　そこはせめて、おねーさんと言え！」="M-Moth--!? Atleast say ""elder sister"" instead!"
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c31/15/70/translation.txt:6]: Quote issues

**Review by GeBo1 — CHANGES_REQUESTED** [review]
Only got through the first 80 files or so, but there seems to be issues with lines with `"` in them.

They come out like this:
```
=""blah blah," he said."
```
Rather than:
```
="blah blah," he said.
```
 
Also if it's simple enough could you restore the line of whitespace separating the `CHOICE:` entries so the diffs line up?

**Comment by ManlyMarco** [comment]
@GeBo1 Fixed both of these issues, looks like it's ready to merge.
Adding extra lines around CHOICE: caused some other diff noise but it seems to be much less than before.

**Pull request merged**